gfortran_utils.sh

```diff
@@ -97,6 +97,21 @@ if [ "$(uname)" == "Darwin" ]; then
         hdiutil attach -mountpoint /Volumes/gfortran $GFORTRAN_DMG
         sudo installer -pkg /Volumes/gfortran/gfortran.pkg -target /
         check_gfortran
+        if [[ "${PLAT:-}" == "universal2" || "${PLAT:-}" == "arm64" ]]; then
+            curl -L -O https://github.com/isuruf/gcc/releases/download/gcc-10-arm-20210112/gfortran-darwin-arm64.tar.gz
```

Comment:
Maybe encapsulate this as its own function - like `install_arm64_gfortran`?

Reply:
Done